content/strategy-goals/strategy/code-graph/code-insights/index.md

```diff
@@ -157,9 +157,9 @@ The next six months are focused on building a product that can quickly prove out
 
 ### Goals
 
-#### FY22q4
+#### FY23Q1
```

Comment:
Minor thing but I think we can remove this heading. The reader can assume whatever is here are the current goals, and there isn't any other section for goals in other quarters.

Reply:
I think clarity here is helpful. If we removed it, while the reader can assume the goals are current, it isn't clear what scope "current" refers to. E.g. it could be the current quarter, but it could also be the current financial year.

Reply:
Then let's say something like "We plan to complete these goals by X." Writing down the name of the current quarter as a section heading is less clear/useful.

Reply:
```suggestion
These are the goals we plan to fully complete by the end of Q1 FY23.
```